dapps/marketplace/src/pages/settings/Settings.js

```diff
@@ -308,6 +309,22 @@ class Settings extends Component {
                           />
                         </div>
                       </div>
+                      <div className="form-group row">
+                        <div className="col-sm">
+                          <label htmlFor="indexing">
```

Comment:
nit: should this label be changed to "growth" rather than indexing ?

Reply:
Actually, there is no element with `id="indexing"` or `id="growth"` props as far as I can see. So this `htmlFor` prop has no effect

Reply:
We can probably get rid of all `htmlFor` attributes or set an unique `id` for every input (and change its label's `htmlFor` attribute accordingly).